awesome_cart/power.py

```diff
@@ -40,6 +41,10 @@ def get_power_user_settings():
 
 	user_doc = frappe.get_doc("User", frappe.session.user)
 
+	# pass quotation with response object to be recorded by FullStory
+	session_quotation = get_user_quotation(get_awc_session())
```

Comment:
Does this get the entire quotation or just the name?

Reply:
entire quotation, should i just send the 2 fields i we need?